sajibportfolio.html

```diff
@@ -1,289 +1,294 @@
-          experience! If you like what you see and have a project you need
-          coded, don’t hestiate to contact me.
-        </p>
-        <div class="smd">
-          <div class="st">
-            <h2 class="stt">LET’S CONNECT</h2>
-          </div>
-          <ul class="fra">
-            <li class="nb1">
-              <a href="https://github.com/sajibarafatsiddiqui/" target="_blank"><img src="statics/images/github.png"
-                  alt="github profilelink" /></a>
-            </li>
-            <li class="nb1">
-              <a href="https://www.linkedin.com/in/sajib-arafat-siddiqui-7aa8b366" target="_blank"><img
-                  src="statics/images/Linkedin_icon.png" alt="linkedin profile link" /></a>
-            </li>
-            <li class="nb1">
-              <a href="#" target="_blank"><img src="statics/images/doits.png" alt="angel list link" /></a>
-            </li>
-            <li class="nb1">
-              <a href="#" target="_blank"><img src="statics/images/twitter.png" alt="angel list link" /></a>
-            </li>
-            <li class="nb1">
-              <a href="#" target="_blank"><img src="statics/images/M.png" alt="angel list link" /></a>
-            </li>
-          </ul>
-          <button class="rsm"><a class="brt" target="_blank" href="#">Get my resume</a></button>
+  </section>
+
+  <section id="about_me">
+    <div class="top">
+      <h2 class="pta1">
+        About <br />
+        Myself
+      </h2>
+      <p class="pta2">
+        Hello I’m a data Scientist and software developer! I can help you
+        build a product , feature or website Look through some of my work and
+        experience! If you like what you see and have a project you need
+        coded, don’t hestiate to contact me.
+      </p>
+      <div class="smd">
+        <div class="st">
+          <h2 class="stt">LET’S CONNECT</h2>
         </div>
-  
+        <ul class="fra">
+          <li class="nb1">
+            <a href="https://github.com/sajibarafatsiddiqui/" target="_blank"><img src="statics/images/github.png"
+                alt="github profilelink" /></a>
+          </li>
+          <li class="nb1">
+            <a href="https://www.linkedin.com/in/sajib-arafat-siddiqui-7aa8b366" target="_blank"><img
+                src="statics/images/Linkedin_icon.png" alt="linkedin profile link" /></a>
+          </li>
+          <li class="nb1">
+            <a href="#" target="_blank"><img src="statics/images/doits.png" alt="angel list link" /></a>
+          </li>
+          <li class="nb1">
+            <a href="#" target="_blank"><img src="statics/images/twitter.png" alt="angel list link" /></a>
+          </li>
+          <li class="nb1">
+            <a href="#" target="_blank"><img src="statics/images/M.png" alt="angel list link" /></a>
+          </li>
+        </ul>
+        <button class="rsm"><a class="brt" target="_blank" href="#">Get my resume</a></button>
```

Comment:
- [ ] As requested earlier. please implement the hover functionality for all your buttons, see design :point_down: 
![image](https://user-images.githubusercontent.com/97521557/206856668-6e98cc87-42fd-4cea-8038-fc376de24192.png)

Reply:
applied. Thanks